main.go

```diff
@@ -155,8 +155,8 @@ func filterSigningFiles(fileList []string) []string {
 	return signingFiles
 }
 
-func removeFilesFromBuildArtifact(aapt, pth string, files []string) error {
-	cmdSlice := append([]string{aapt, "remove", pth}, files...)
+func removeFilesFromBuildArtifact(pth string, files []string) error {
+	cmdSlice := append([]string{"zip", "-d", pth}, files...)
```

Comment:
For consistency, it would be nice to print the full path of the zip binary in the command, or at the beginning of the step using exec.LookPath. (Also aapt path is printed, but it is not used anymore)